raiden/transfer/channel.py

```diff
@@ -1354,6 +1524,47 @@ def events_for_close(
     return events
 
 
+def events_for_expired_lock(
+        channel_state: NettingChannelState,
+        locked_lock: LockedTransferUnsignedState,
+        pseudo_random_generator: random.Random,
+):
+    nonce = get_next_nonce(channel_state.our_state)
+    locked_amount = get_amount_locked(channel_state.our_state)
```

Comment:
where is the lock being removed?

Reply:
I updated this function's code to also remove the lock.